packages/validator/src/cli-validator/run-validator.js

```diff
@@ -266,13 +266,9 @@ async function runValidator(cliArgs, parseOptions = {}) {
       const numWarnings = results.warning.summary.total;
       if (numWarnings > limitsObject.warnings) {
         exitCode = 1;
-        // add the exceeded warnings limit as an error
-        results.error.results.push({
-          line: 0,
-          rule: 'warnings-limit',
-          path: [],
-          message: `Number of warnings (${numWarnings}) exceeds warnings limit (${limitsObject.warnings}).`
-        });
```

Comment:
I wasn't a fan of the fact that we were adding a made-up rule here to signal the limit being exceeding (though it was almost certainly my idea a few years ago 🙂 )

I think this is more simple and direct